### PR TITLE
stop_serving method

### PR DESCRIPTION
**Pull request by Jacotsu**
This pull request implements the `stop_serving` method, which enables graceful halting of bots using webhooks

### CODE REVIEW

File: src/tourmaline/client/webhook_methods.cr [hunk 1]
```diff
@@ -20,6 +20,8 @@ module Tourmaline
           end
         end
 
+        server = @webhook_server.not_nil!
```
Comment (watzon) [inline src/tourmaline/client/webhook_methods.cr:23]: You could avoid this by changing like 12 to:

```crystal
@webhook_server = server = HTTP::Server.new do |context|`
```

**Review by watzon — CHANGES_REQUESTED** [review]
Just one small thing and I'll get this merged

File: src/tourmaline/client/webhook_methods.cr (hunk 1 above)
Comment (Jacotsu) [reply src/tourmaline/client/webhook_methods.cr]: Fixed in latest commit

**Review by watzon — APPROVED** [review]

**Comment by watzon** [comment]
Thanks!

**Pull request merged**